pkgs/development/python-modules/ha-hoymiles-wifi/default.nix

```
@@ -0,0 +1,30 @@
+{
+  pkgs,
+  lib,
+  fetchFromGitHub,
+  buildHomeAssistantComponent,
+}:
+
+buildHomeAssistantComponent rec {
+  pname = "ha-hoymiles-wifi";
+  owner = "suaveolent";
+  domain = "hoymiles_wifi";
+  version = "0.5.1";
+
+  src = fetchFromGitHub {
+    owner = "suaveolent";
+    repo = "ha-hoymiles-wifi";
+    tag = "v${version}";
+    hash = "sha256-6NxsnRAo8KjlKYfyqosdS0Q34j0KBNNRUWbZmQOvxJk=";
+  };
+
+  dependencies = [ pkgs.python3Packages.hoymiles_wifi ];
```

```suggestion
  dependencies = [ hoymiles-wifi ];
```

Consume this directly as a function argument.